bin/link_check.py

```diff
@@ -165,11 +165,17 @@ def test_links(self, new_url, urls):
                          'status': -1,
                          'resources': new_url[url]}
                 else:
-                    res={'timestamp': now,
-                         'status':response.status_code}
-                    if response.status_code != requests.codes.ok:
-                        res['resources'] = new_url[url]
-                        res['org'] = urls.get(url, None)
+                    try:
+                        res={'timestamp': now,
+                             'status':response.status_code}
+                        if response.status_code != requests.codes.ok:
+                            res['resources'] = new_url[url]
+                            res['org'] = urls.get(url, None)
+                    except:
```

Comment:
I can not remember the exact error. It is not an exception type. I believe it is from the test_ftp(), since last commit.